BrowserKit/Tests/SiteImageViewTests/SVGImageProcessorTests.swift

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+@testable import SiteImageView
+import SwiftDraw
+import Kingfisher
+
+class SVGImageProcessorTests: XCTestCase {
+    let svgFaviconURL = URL(string: "https://news.ycombinator.com/y18.svg")!
+    let icoFaviconURL = URL(string: "https://www.mozilla.org/favicon.ico")!
```

Comment:
In theory this is fine, however when any of these websites fail or go down our tests won't work. 
One way to achieve this is to save these locally or run it via GCDWebServer so in theory you are still "Downloading" but via gcdwebserver.

Reply:
Or you could mock it for better reliability

Reply:
Heh I knew this was going to come up so I was planning to ask what you thought was the best way. Ideally I'd like to test with Kingfisher itself for this so mocking the server request is probably the right way to go.

I see a few examples where we use `GCDWebServer` so I'll update our tests to use that. 👍

Reply:
For the last test I was having trouble bundling an SVG into the package's test target. In the end I just hardcoded the image fetch so I could get this PR up as I was spending too much time on it. I was trying to retrieve an SVG image resource using `Bundle.module` as per these [Apple Docs](https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package). Is that something you've done before?

Reply:
Ha! you are well prepared for that answer. 

Okay regarding SVG, hmm I would assume its same as adding a file to our XCode project and retrieve it using Bundle.module

We do something similar with the .json files. Let me give you an example for how we do it in UITest

https://github.com/mozilla-mobile/firefox-ios/blob/main/firefox-ios/Client/Application/UITestAppDelegate.swift#L295-L302

Basically you won't be able to use SVG but I am sure this will get you the file

Reply:
Haha! 😂  Figured I'd get some good tips in the review!

Ok thanks for the example. Once I'm done my current task I'll switch back to this and give it a try. 🙏 

I was seeing lots of stuff about adding a resources folder to the Package dependencies so I'm hoping that won't be necessary.